.github/workflows/master.yaml

```diff
@@ -20,11 +20,9 @@ jobs:
       - run: go version
       - run: make deps
       - run: make check-fmt
-      - run: make build
       - run: make vet
       - run: make staticcheck
       - run: make check-race
-      - run: make cicheck
```

Comment:
There is make check-race before

Reply:
So, this step is valid, but redundant, right?

Reply:
yes